qemu/tests/qemu_disk_img_convert.py

```diff
@@ -33,6 +36,28 @@ def convert(self, t_params=None):
         self.trash.append(converted)
         return params
 
+    @error_context.context_aware
```

Comment:
For default mode and strict mode, test steps are the same before comparing. After we simply specify "compare mode" in variants, It is a waste of time when both default mode and strict mode are tested. However, compare_mode_list can solve this issue by specify different mode like compare_mode_list="default", compare_mode_list="strict" or compare_mode_list="default strict"